common/history/characters/0_dvg_tawantinsuyu.txt

```diff
@@ -69,5 +72,22 @@
 #				exile_character = yes
 #			}
 		}
+		create_character = {
```

Comment:
[10:25:33][jomini_script_system.cpp:263]: Script system error!
  Error: Failed to scope to culture by key 'Japanese'
  Script location: file: common/history/characters/0_dvg_tawantinsuyu.txt line: 79

[10:25:33][jomini_script_system.cpp:263]: Script system error!
  Error: Event target link 'cu' returned an unset scope
  Script location: file: common/history/characters/0_dvg_tawantinsuyu.txt line: 79